README.md

```diff
@@ -1,12 +1,19 @@
-# Nuclear Speckle Image Analysis
+# Nuclear Speckle Image Profiling
 
-In this repository, image analysis and image-based profiling is performed on a nuclear speckle dataset to extract CellProfiler morphology features to predict nuclear speckle protein markers.
+In this repository, image analysis and image-based profiling is performed on a nuclear speckle dataset to extract CellProfiler morphology features for predicting nuclear speckle protein markers.
 
 We have 3 channels in this assay:
 
-**DAPI** - nucleus stain
-**A647** - SON protein marker for nuclear speckles
-**GOLD** - SRRM2 protein marker for nuclear speckles
+- **DAPI** - nucleus stain
+- **A647** - SON protein marker for nuclear speckles
+- **GOLD** - SRRM2 protein marker for nuclear speckles
```

Comment:
Consider adding the full fluorophore name and a small one-sentence blurb about what the protein's function if or where it binds in the nucleus.

Reply:
These are the staining names that I have been provided, I am happy to look into more about these proteins. This might need to wait for another PR once we meet with our collaborator. Thanks for the suggestion!